src/test_sort_merge.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+"""Test file for merge sort."""
+
+LIST_NUM_SORTED = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+LIST_NUM_SORTED2 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+LIST_NUM_UNSORTED = [6, 2, 7, 1, 5, 3, 8, 4, 9, 0]
+LIST_NUM_REVERSE = LIST_NUM_SORTED[::-1]
+
```

Comment:
You have your best and worst cases, but what about your edge cases? A list of strings? negative numbers? Repeat numbers? Testing for stability? Empty lists? You need to handle these cases.